js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetNode.tsx

```diff
@@ -28,14 +28,16 @@ interface Props {
 }
 
 export const AssetNode = React.memo(({definition, selected}: Props) => {
-  const displayName = definition.assetKey.path[definition.assetKey.path.length - 1]!;
   const isSource = definition.isSource;
 
   const {liveData} = useAssetLiveData(definition.assetKey);
 
   return (
     <AssetInsetForHoverEffect>
-      <Box flex={{direction: 'row', justifyContent: 'space-between', alignItems: 'center'}}>
+      <Box
+        flex={{direction: 'row', justifyContent: 'space-between', alignItems: 'center'}}
+        style={{minHeight: 24}}
```

Comment:
At some point this changed and the assets now shift up when there are no reason tags, which makes the arrows misaligned

Reply:
oof thats on me. It changed last week!

Reply:
Should we change it back?

Reply:
Ahh it's ok! I think adding this min-height does the trick 🙏